packages/admin-panel/src/VizBuilderApp/components/PreviewOptions.js

```diff
@@ -151,6 +173,24 @@ export const PreviewOptions = () => {
     setLocation(value.code);
   };
 
+  const handleChangeStartDate = date => {
+    if (!date) {
```

Comment:
I suggest to set value when it is null. User case is user wants to see all data from all period, they will delete the previous date input.